### PR TITLE
update loguniform for smac

### PR DESCRIPTION
**Pull request by Crysple**
since loguniform in smac behaves differently, it is necessary to unify the behaviors of all tuner's loguniform.

### CODE REVIEW

File: src/sdk/pynni/nni/smac_tuner/convert_ss_to_scenario.py [hunk 1]
```diff
@@ -63,7 +64,8 @@ def generate_pcs(nni_search_space_content):
                                 json.dumps(search_space[key]['_value']),
                                 json.dumps(search_space[key]['_value'][0])))
                         elif search_space[key]['_type'] == 'loguniform':
-                            pcs_fd.write('%s real %s [%s] log\n' % (
+                            search_space[key]['_value'] = list(np.round(np.log(search_space[key]['_value']), 10))
```
Comment (QuanluZhang) [inline src/sdk/pynni/nni/smac_tuner/convert_ss_to_scenario.py]: would be better to briefly explain why you make this change.

Comment (QuanluZhang) [reply src/sdk/pynni/nni/smac_tuner/convert_ss_to_scenario.py]: I mean np.round

File: src/sdk/pynni/nni/smac_tuner/smac_tuner.py [hunk 2]
```diff
@@ -170,13 +172,19 @@ def generate_multiple_parameters(self, parameter_id_list):
         '''
```
Comment (QuanluZhang) [inline src/sdk/pynni/nni/smac_tuner/smac_tuner.py]: should also update the function `generate_parameters`.

File: src/sdk/pynni/nni/smac_tuner/convert_ss_to_scenario.py (hunk 1 above)
Comment (Crysple) [reply src/sdk/pynni/nni/smac_tuner/convert_ss_to_scenario.py]: okay, I will add some comments

File: src/sdk/pynni/nni/smac_tuner/smac_tuner.py (hunk 2 above)
Comment (Crysple) [reply src/sdk/pynni/nni/smac_tuner/smac_tuner.py]: I will add it. But it seems that nni will never invoke generate_parameters actually if you have overwrite generate_parameters already

Comment (QuanluZhang) [reply src/sdk/pynni/nni/smac_tuner/smac_tuner.py]: yes, but still we should make `generate_parameters` correct.

**Pull request merged**